packages/schema/src/config/app.ts

```diff
@@ -243,7 +243,7 @@ export default defineUntypedSchema({
    * @type {string | boolean}
    */
   spaLoadingTemplate: {
-    $resolve: async (val, get) => typeof val === 'string' ? resolve(await get('srcDir'), val) : val
+    $resolve: async (val, get) => typeof val === 'string' ? resolve(await get('srcDir'), val) : val ?? null
```

Comment:
Would `undefined` work equally same as `null` here? (if so, it might be better default. null is an evil to introduce bugs!)

Reply:
No, undefined is the issue. untyped resolves it to `{}` in that case.